packages/terra-navigation-side-menu/src/clinical-lowlight-theme/NavigationSideMenu.module.scss

```diff
@@ -0,0 +1,64 @@
+:local {
+  .orion-fusion-theme {
+    --terra-navigation-side-menu-action-header-background-color: #141719;
```

Comment:
Break the theme variables for side menu and menu item into separate files

Reply:
Done 0a0e490